Add persist package and tool to the release note (#6325)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

- Added support for the new NATS JetStream client with publishing and subscribing functionalities

##### `persist` package

> - Added a new `persist` package to provide the functionality required for Ballerina Persistence Layer. 

Reviewer: ```suggestion
- Added a new `persist` package to provide the functionality required for the Ballerina persistence layer. 
```

>   Ballerina Persistent Layer provides functionality to store and query data conveniently.

Reviewer: ```suggestion
  The Ballerina persistent layer provides the functionality of storing and querying data conveniently.
```

>   This is an Experimental feature; APIs might change in future releases.

Reviewer: ```suggestion
  >**Info:** This is an experimental feature; APIs might change in future releases.
```

#### OpenAPI Tool
Added support to generate Ballerina client and service declarations from Swagger 2.0(i.e. OpenAPI 2.0) definitions

#### Persist Tool

> - Added a new `persist` command to make it easy to enable Ballerina Persistence Layer in a Ballerina project.

Reviewer: ```suggestion
- Added a new `persist` command to make it easy to enable the Ballerina persistence layer in a Ballerina project.
```

>   With this support, users can define an entity data model, validate the model and generate `persist` clients, which provide convenient APIs to store and query data in a data store.

Reviewer: ```suggestion
  With this support, users can define an entity data model, validate the model, and generate `persist` clients. This provides convenient APIs to store and query data in a data store.
```
